### PR TITLE
Improve logging

### LINKED ISSUE #221: Option to reduce blurry image console output
Using the optimizer-express package, every image that is successfully rendered with AddBlurryImagePlaceholders is logged to the console - and with longer image urls creates a lot of noise in the logs. Could we add an option to suppress this output?

eg.
AMP Optimizer Adding blurry image placeholder for (src / resolved): https://www.example.com/longurl/image.jpg https://www.example.com/longurl/image.jpg

### PR DESCRIPTION
**Pull request by sebastianbenz**
Fixes #221

### CODE REVIEW

File: packages/optimizer-express/lib/AmpOptimizeMiddleware.js [hunk 1]
```diff
@@ -133,16 +132,16 @@ class AmpOptimizerMiddleware {
...
+        if (!(options.ampOnly && !isAmp(body))) {
+          try {
             body = await optimizer.transformHtml(body, ampOptimizerParams);
+          } catch (err) {
+            console.error('Error applying AMP Optimizer. Sending original page', err);
           }
-          res.setHeader('Content-Length', Buffer.byteLength(body, 'utf-8'));
-          res.end(body, 'utf-8');
-        } catch (err) {
-          console.error('Error applying AMP Optimizer. Sending original page', err);
```
Comment (sebastianbenz) [inline packages/optimizer-express/lib/AmpOptimizeMiddleware.js]: @andreban Did this work? If yes, why?

Comment (andreban) [reply packages/optimizer-express/lib/AmpOptimizeMiddleware.js]: It did work -> It used to call `res.end()` inside a catch block before it was migrated to async / await in this change: https://github.com/ampproject/amp-toolbox/commit/17856f6133d3662dc9684c32391f47e49bcbeba0#diff-c8065ddd90540be59b6bd37efc257e38

File: packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js [hunk 2]
```diff
@@ -166,14 +165,16 @@ class AddBlurryImagePlaceholders {
           return image.getBase64Async('image/png');
         })
         .then((dataURI) => {
+          log.debug(img.attribs.src);
```
Comment (andreban) [inline packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js]: nit: this is going to print an URL. Maybe it's worth adding more context on what it is?

File: packages/optimizer-express/lib/AmpOptimizeMiddleware.js (hunk 1 above)
Comment (sebastianbenz) [reply packages/optimizer-express/lib/AmpOptimizeMiddleware.js]: Hah!

File: packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js (hunk 2 above)
Comment (sebastianbenz) [reply packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js]: it's going to print: `AMP Optimizer AddBlurryImagePlaceholders bball.jpg`. Should I add more?

Comment (andreban) [reply packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js]: Maybe the URL for the original image (if it's not being logged at some point before this)

Comment (sebastianbenz) [reply packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js]: @andreban tweaked it a bit in the latest commit.

Comment (sebastianbenz) [reply packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js]: Success case:

```
AMP Optimizer AddBlurryImagePlaceholders bball.jpg [SUCCESS]
```

Error case:

```
AMP Optimizer AddBlurryImagePlaceholders ERROR ENOENT: no such file or directory, open '/Users/sbenz/projects/amp-toolbox/packages/optimizer/spec/assets/bball2.jpg'

AMP Optimizer AddBlurryImagePlaceholders bball2.jpg [FAIL]
```

**Pull request merged**